ftw/simplelayout/contenttypes/contents/textblock.py

```diff
@@ -70,6 +76,18 @@ class ITextBlockSchema(form.Schema):
 alsoProvides(ITextBlockSchema, IFormFieldProvider)
 
 
+class TextBlockImageLimitValidator(ImageLimitValidator):
+    """
```

Comment:
Is a docstring definition required here? 🤔 I'd remove the empty string if it does not matter 😉